Gulpfile.js

```diff
@@ -34,7 +34,7 @@ const createPackageFilesForTests    = require('./gulp/helpers/create-package-fil
 const {
     TESTS_GLOB,
     LEGACY_TESTS_GLOB,
-    MULTIPLE_WINDOWS_TESTS_GLOB,
+    MULTIPLE_WINDOWS_TASK_TESTS_SOURCE,
```

Comment:
I think it is better to not modify this task.
I propose creating a separate GitHub task name d Headed browser for all tests that require the headed browser.

Reply:
Yes, it was modified only for one test purpose. I already created a separate task job that will run such a tests.